api/converter/converter_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 	"time"
 
+	time2 "github.com/yorkie-team/yorkie/pkg/document/time"
+
 	"github.com/stretchr/testify/assert"
 
 	"github.com/yorkie-team/yorkie/api/converter"
```

Comment:
I hope we group the package into 3 parts.

- Standard library
- 3rd party library
- internal package

I referenced the [Uber Go Style Guide](https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering).